File: Codechef/Chef_Bottle/Chef_Bottle.cpp
#include <iostream>
using namespace std;

int main()
{
    int T;
    cin >> T;
    while (T--)
    {
        int N, X, K, num;
        cin >> N >> X >> K;
        if (X <= K)
        {
            num = K/X;
            if (num >= N)
            {
                cout << N << endl;
            }
            else
            cout << num << endl;
        }
        else
        {
            cout << K/X << endl;
        }
    }
    return 0;
}

File: Codechef/Chef_Bottle/Chef_Bottle.py
T = int(input())
while T:
    T -= 1
    N, X, K = map(int, input().split())
    if X <= K:
        num = K//X
        if (num >= N):
            print(N)
        else:
            print(num)
    
    else:
        print(K//X)